src/battle_util.c

```diff
@@ -9444,22 +9444,22 @@ static inline u32 CalcMoveBasePowerAfterModifiers(struct DamageCalculationData *
         break;
     case ABILITY_PIXILATE:
         if (moveType == TYPE_FAIRY && gBattleStruct->ateBoost[battlerAtk])
-            modifier = uq4_12_multiply(modifier, UQ_4_12(1.2));
+            modifier = GetGenConfig(GEN_CONFIG_ATE_MULTIPLIER) >= GEN_7 ? uq4_12_multiply(modifier, UQ_4_12(1.2)) : uq4_12_multiply(modifier, UQ_4_12(1.3));
```

Comment:
I think this is slightly less verbose?
```suggestion
            modifier = uq4_12_multiply(modifier, UQ_4_12(GetGenConfig(GEN_CONFIG_ATE_MULTIPLIER) >= GEN_7 ? 1.2 : 1.3));
```

Alternatively, maybe a `GetAteAbilityModifier` would be better?

Reply:
I'm not sure that `GetAteAbilityModifier` is super necessary when this only comes up in exactly one place in the code for each ability, and these can be one-liner'd cleanly. I'll add it if you think it's important but I don't think it's necessary.

Definitely agree this is less verbose, pushed other commits so can't use the suggestion but have pushed this change :)